myuw/static/vue/components/home/applicant/seattle.vue

```diff
@@ -87,24 +89,24 @@
       </h5>
       <ul class="list-unstyled myuw-text-md">
         <li v-if="applicantData.is_freshman">
-          <a href="https://admit.washington.edu/apply/dates-deadlines/"
-             target="_blank" label="Key dates &amp; deadlines for freshmen"
+          <a v-out="'Key dates deadlines for freshmen'"
```

Comment:
Why do we provide a v-out value here? we can use the innerText

Reply:
Thanks!